tests/hypothesis_strategies/kernels.py

```diff
@@ -12,3 +12,16 @@
     lengthscale_prior=st.one_of(st.none(), priors),
 )
 """A strategy that generates Matern kernels."""
+
+base_kernels = st.one_of([matern_kernels])
```

Comment:
For now, this suffices

```suggestion
base_kernels = matern_kernels
```